tidb-cloud/tidb-cloud-billing.md

```diff
@@ -188,4 +188,56 @@ If you have agreed with our sales on a contract and received an email to review
 3. Click **Contract**. The contract list is displayed.
 4. Click **Download**, **Accept**, or **Reject** according to your need.
 
-To learn more about contracts, feel free to [contact our sales](https://www.pingcap.com/contact-us/).
\ No newline at end of file
+To learn more about contracts, feel free to [contact our sales](https://www.pingcap.com/contact-us/).
+
+## Billing from AWS Marketplace or Google Cloud Marketplace
+
+If you are the owner of an organization, you can link your AWS billing account or Google Cloud billing account to TiDB Cloud. Otherwise, skip this section.
```

Comment:
```suggestion
If you are the owner or billing administrator of your organization, you can link your AWS billing account or Google Cloud billing account to TiDB Cloud. Otherwise, skip this section.
```